### ISSUE TITLE
The maps turn white

### ISSUE THREAD

**Issue by PJGamerr** [SydMontague/ImageMaps#49]
the plugin works perfectly .. puts the perfect maps .. but I don't know why after a few days that the map is on the wall it becomes invisible .. white .. clicking with the mouse wheel gives me an empty map .. I don't understand the reason, could you help me? is very important

**Comment by SydMontague** [comment]
Is there anything in your server logs? Are the backing images files still in the images folder? Are there any other plugins that could interfere with my plugin?

**Comment by PJGamerr** [comment]
yes in the plugin folder there are still the references of the various maps, in the logs there are no problems .. as I said above this problem does not arise immediately but after a few days, I put my maps on a wall about 4 days ago and today totally randomly they have become empty .. the maps are still there but they are empty .. I don't understand .. then

**Comment by Tsoccerguy3** [comment]
This happened to our server as well after the update to 1.18.1 , We just assumed it was a height related issue on how Paper was processing format 8 chunks . 

To the developer. 
Check for 1.18.1 changes to functions and methods in the source . Also update Java 17.0.1 and other dependencies as a caution , having to do with security issues that surfaced last week.

**Comment by SydMontague** [comment]
The plugin works fine on my local server using both Spigot and Paper 1.18.1, I can't reproduce the problem

It also doesn't use log4j directly, so as long as the server itself is patched it shouldn't apply.

**Comment by mercurialmusic** [comment]
I'm also getting this for the first time after the update to 1.19.1. It's not the entire image, but maybe a couple maps within it will go invisible. No errors, reloading the image seems to fix it at least temporarily even in the existing maps. Doesn't seem to matter whether the map is new or old, it's even happening to newly placed maps, after a restart one or two might go wonky. Very strange.

**Comment by Kadeluxe** [comment]
This is an old issue in Minecraft / Spigot server. I've experienced maps corruption with such plugins even back in 1.14. No idea why and how to reproduce it reliably but sadly it happens.
